### PR TITLE
Membership page image

### PR DESCRIPTION
**Pull request by jtaaa**
Added "Are You A Member" image with animation to membership page
AreYouAMember

## Notes
I need to add the animations for the header separately. Any content that comes below the "Are You A Member" image should also have an enter animation with a timeout of 1s so that the image doesn't cover it.

### CODE REVIEW

**Comment by jtaaa** [comment]
For now it looks fine up to 1440px desktop but on larger screens, the image is really small. I don't think the solution is to make the image bigger though as it will lose quality.

**Comment by jtaaa** [comment]
Also the image is huge... almost 200kb but we can optimize after if it is a problem

File: client/src/components/ShrinkImage/index.tsx [hunk 1]
```diff
@@ -15,10 +15,13 @@ const styles = (theme: Theme) =>
...
       height: '100%',
-      maxWidth: '80vw',
-      maxHeight: '80vh',
+      maxWidth: '75vw',
+      maxHeight: '75vh',
       margin: `${theme.spacing(4)}px auto`,
       transition: 'transform 300ms ease-in-out',
+      [theme.breakpoints.down('sm')]: {
+        height: 'auto',
+      },
```
Comment (jtaaa) [inline client/src/components/ShrinkImage/index.tsx:24]: On desktop we can control the size of the image by setting the height of the containing div. We can make it bigger or smaller without hitting the maxWidth and messing up the width/height ratio.

On mobile we let the image scale down appropriately to fit 75% the width of the screen.

Comment (chriskrogh) [reply client/src/components/ShrinkImage/index.tsx:24]: 👍

**Comment by chriskrogh** [comment]
> For now it looks fine up to 1440px desktop but on larger screens, the image is really small. I don't think the solution is to make the image bigger though as it will lose quality.

It looks okay in 4k so this is fine

**Pull request merged**